### DISCUSSION TITLE
New install returning 502 bad gateway

### DISCUSSION THREAD

**Discussion by coolcat818382901** [kimai/kimai#5028] [Questions & Answers]
I installed Kimai, using php8.1.2 because 8.3 was unavailable via my pkg manager. Everything went through, but I'm receiving a 502 Bad Gateway via Cloudflare. I set the SSL settings properly, and Let's Encrypt. There's no indication anywhere that the install failed.

**Comment by coolcat818382901** [comment]
Turns out you needed a PPA to install PHP 8.3. The docs of Kimai are a bit out of touched of what's needed (from what I can tell??) - tried on ubuntu and debian with the same result of installing 8.3.

For posterity purposes and for those who need it on ubuntu add this ppa:
```
add-apt-repository ppa:ondrej/php
```

**Reply by kevinpapst** [reply]
> tried on ubuntu and debian

Which ones? 
The last time I tried, 8.3 was included on the Ubuntu repos.

**Reply by kevinpapst** [reply]
Compare 20.04: https://www.kimai.org/documentation/fresh-ubuntu-20.html (ondrej repo)
With 22.04: https://www.kimai.org/documentation/fresh-ubuntu-22.html (no additional repo)
